File: fft/test_fft.m
fname = importdata("data/fname.txt");
for k = 1:numel(fname)
	fprintf("%20s\t",fname{k})
	x = load(fname{k});
	tic
	y = fft(x);
	xx = ifft(x);
	t = toc;
    fprintf("%12d (ns)\n",round( 1e9 * t ))
end

File: fft/test_fft.py
import numpy as np
import time

# open data
f = open("data/fname.txt","r")
fname = f.readlines()
f.close()

for s in fname:

	s = s.replace("\n","")
	x = np.loadtxt(s)
	t = - time.time_ns()

	# the test
	y = np.fft.fft(x)
	xx = np.fft.ifft(y)

	t += time.time_ns()
	print("%20s\t%d"%(s,t))